App/src/pages/dossier/DossierReview.tsx

```diff
@@ -0,0 +1,466 @@
+                                                {courseCreationRequest.newCourse?.subject}{" "}
+                                                {courseCreationRequest.newCourse?.catalog}
+                                            </Link>
+                                        </Text>
+                                    ))}
+                                </Stack>
+                                <Stack mb={4}>
+                                    <Heading size={"md"} color="brandBlue">
+                                        Course Modification Requests:
+                                    </Heading>
+                                    {dossierDetails?.courseModificationRequests?.map((courseModificationRequest) => (
+                                        <Text key={courseModificationRequest.id} as="u">
+                                            <Link to={BaseRoutes.ManageableGroup}>
+                                                {courseModificationRequest.course?.subject}{" "}
+                                                {courseModificationRequest.course?.catalog}
+                                            </Link>
+                                        </Text>
+                                    ))}
+                                </Stack>
+                                <Stack mb={4}>
+                                    <Heading size={"md"} color="brandBlue">
+                                        Course Deletion Requests:
+                                    </Heading>
+                                    {dossierDetails?.courseDeletionRequests?.map((courseDeletionRequest) => (
+                                        <Text key={courseDeletionRequest.id} as="u">
+                                            <Link to={BaseRoutes.ManageableGroup}>
+                                                {courseDeletionRequest.course?.subject}{" "}
+                                                {courseDeletionRequest.course?.catalog}
+                                            </Link>
+                                        </Text>
+                                    ))}
+                                </Stack>
+                            </Stack>
+                        </Stack>
+                        <Stack w="75%" p={8}>
+                            <Stack>
+                                <Center>
+                                    <Heading as="h2" size="xl" color="brandRed">
+                                        <Text align="center">Add Message</Text>
+                                    </Heading>
+                                </Center>
+                                <Stack mb={2}>
+                                    <Text as="span">
+                                        ***You are a member of the{" "}
+                                        <Text as="span" fontWeight={"bold"} fontSize={20}>
+                                            Gina Cody School Faculty Council
```

Comment:
When backend functionality is created, this group name should always be the name of the currently reviewing group.

Reply:
So does this mean that other people from other groups cannot review this Dossier at the same time?
For instance, if the currently reviewing group is Department Council, then someone else from the Gina Cody School Faculty Council cannot review this Dossier?